lib/bootloader_setup.pm

```diff
@@ -581,6 +583,21 @@ sub bootmenu_default_params {
         }
         push @params, "Y2DEBUG=1";
     }
+    elsif (get_var('AGAMA')) {
+        wait_screen_change { send_key "e" };
+        send_key "down";
+        send_key "down";
+        send_key "down";
+        send_key "down";
+        wait_screen_change { send_key "end" };
```

Comment:
It would be probably more reliable to use needles here instead of `wait_screen_change`.

Reply:
ufff this would be a needle for a wrapped text in grub2 edit. I did copy same approach as from one of other images.